lms/templates/courseware/progress.html

```diff
@@ -82,30 +84,31 @@ <h4 class="hd hd-4 title">${certificate_data.title | h}</h4>
                     <h3 class="hd hd-4 eligibility-heading">${_("Requirements for Course Credit")}</h3>
                     <div class="credit-eligibility-container">
                         %if credit_course_requirements['eligibility_status'] == 'not_eligible':
-                        <span class="eligibility_msg">${_("{student_name}, you are no longer eligible for credit in this course.").format(student_name=student.profile.name) | h}</span>
+                        <span class="eligibility_msg">${_("{student_name}, you are no longer eligible for credit in this course.").format(student_name=student.profile.name)}</span>
                         %elif credit_course_requirements['eligibility_status'] == 'eligible':
-                        <span class="eligibility_msg">${_("{student_name}, you have met the requirements for credit in this course.").format(student_name=student.profile.name) | h}
-                            ${_("{a_start}Go to your dashboard{a_end} to purchase course credit.").format(
-                                a_start=u"<a href={url}>".format(url=reverse('dashboard')),
-                                a_end="</a>"
+                        <span class="eligibility_msg">
+                            ${Text(_("{student_name}, you have met the requirements for credit in this course. {a_start}Go to your dashboard{a_end} to purchase course credit.")).format(
```

Comment:
I tested this manuallhy by commenting out the conditionals so that the text appeared.